src/main/java/seedu/address/model/student/Star.java

```diff
@@ -26,11 +28,18 @@ public Star(Integer numOfStars) {
         this.numOfStars = numOfStars;
     }
 
+    /**
+     * Constructs a star of NO_STAR and to allow editing of stars to 0 in EditCommand.
+     */
+    public Star() {
+        this.numOfStars = 0;
+    }
+
     /**
      * Returns true if a given string is a valid number.
      */
     public static boolean isValidStar(Integer numOfStars) {
```

Comment:
wondering if this is the right name since a star with 0 is valid when instantiating. perhaps it should be isValidStarIncrement?